### PR TITLE
docs: Describe edge-service components

### LINKED ISSUE #32: Release 0.1.1
- [x] Add documentation for 0.1.1
- [ ] Release 0.1.1

### PR DESCRIPTION
**Pull request by talwinder50**
Signed-off-by: talwinder.kaur <talwinder.kaur@securekey.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,3 +5,217 @@
...
+# run linter checks
+make checks
+# run unit tests
+make unit-test
+```
+
+#Dependency
+- [EDV](https://github.com/trustbloc/edv)
+
+# edge-service
```
Comment (fqutishat) [inline README.md]: move it to subfolders like https://github.com/trustbloc/edge-sandbox/tree/master/docs

File: README.md [hunk 2]
```diff
@@ -5,3 +5,217 @@
...
+Issuer presents the data fetched from CMS, [edge-sandbox](https://github.com/trustbloc/edge-sandbox) for the following
+operations: 
+
+- Create issuer profile
+- Create verifiable Credential by using [aries-framework-go](https://github.com/hyperledger/aries-framework-go/tree/master/pkg/doc/verifiable) VC parser
+- Store verifiable credential in [EDV](https://github.com/trustbloc/edv)
+- Retrieve verifiable credential from c
+- Verify verifiable credential 
+
+## REST API ENDPOINTS
```
Comment (fqutishat) [inline README.md]: move it to subfolders like https://github.com/trustbloc/edge-sandbox/tree/master/docs

File: README.md [hunk 3]
```diff
@@ -5,3 +5,217 @@
...
+```
+# run checks and unit tests
+make all
+# run linter checks
+make checks
+# run unit tests
+make unit-test
+```
+
+#Dependency
```
Comment (fqutishat) [inline README.md]: not needed

Comment (talwinder50) [reply README.md]: done

File: README.md [hunk 4]
```diff
@@ -5,3 +5,41 @@
...
+
+Edge Service is a reference implementation that demonstrates the following story:
+
+Issuer presents the data fetched from CMS, [edge-sandbox](https://github.com/trustbloc/edge-sandbox) for the following
+operations: 
+
+- Create issuer profile
+- Create verifiable Credential by using [aries-framework-go](https://github.com/hyperledger/aries-framework-go/tree/master/pkg/doc/verifiable) VC parser
+- Store verifiable credential in [EDV](https://github.com/trustbloc/edv)
+- Retrieve verifiable credential from c
```
Comment (troyronda) [inline README.md]: from c?

File: docs/vc-rest/README.md [hunk 5]
```diff
@@ -0,0 +1,175 @@
+## REST API ENDPOINTS 
+
+### Create issuer profile  - POST /profile
+This is the first endpoint to continue the flow the in the edge-service. 
+Mandatory fields: name, did, uri, signatureType, creator
```
Comment (troyronda) [inline docs/vc-rest/README.md]: not all the fields are obvious from the description. suggest to define (e.g., name).

File: README.md (hunk 4 above)
Comment (talwinder50) [reply README.md]: updated

File: docs/vc-rest/README.md (hunk 5 above)
Comment (talwinder50) [reply docs/vc-rest/README.md]: updated

**Pull request merged**